src/main/kotlin/frc/robot/subsystems/intake/extender/Extender.kt

```diff
@@ -29,14 +31,22 @@ class Extender(private val io: ExtenderIO) : SubsystemBase() {
 
     private var finishedResettingFlag = false
 
-    private fun setPosition(position: Positions): Command =
-        runOnce {
-                io.setPosition(position.position)
-                setpoint = position.position
-                setpointName = position.getLoggingName()
-            }
+    private fun setPosition(position: () -> Distance): Command =
+        sequence(
+                runOnce {
+                    setpoint = position.invoke()
+                    io.setPosition(position.invoke())
```

Comment:
```suggestion
                    setpoint = position.invoke()
                    io.setPosition(setpoint)
```
While this probably won't be a problem in this case. it's better not to invoke a supplier twice, because if the value changes between the two invocations, you might get different values.